BLSQ/iaso
# IA-2765: Update default form for Region/District/Area levels

Related JIRA tickets : [IA-2765](https://bluesquare.atlassian.net/browse/IA-2765)

## Self proofreading checklist



## Changes

Updated the script to create default form for org units other than health facilities!

## How to test
From setuper folder, launch `python3 setuper.py` to create data and open forms list page : [localhost:8081/dashboard/forms/list](http://localhost:8081/dashboard/forms/list)

You should see in the forms list a row with : **Equipment/Pop/Social mob./Microplans** . It's a form for all org unit types other than 
health facility!

## Print screen / video

https://github.com/BLSQ/iaso/assets/6383219/5584f8ed-c3c7-4de2-9297-b7bf97fbb8a4



[IA-2765]: https://bluesquare.atlassian.net/browse/IA-2765?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
